### PR TITLE
Load all articles redux

### PR DESCRIPTION
**Pull request by giorgiosironi**

### CODE REVIEW

File: scripts/load_all_articles.php [hunk 1]
```diff
@@ -50,6 +50,15 @@
     //echo "Count: $articlesCount", PHP_EOL;
     //echo 'Memory: ', memory_get_usage(true), ' bytes', PHP_EOL;
 }
+foreach ($articleIds as $id) {
+    $article = $articles->get($id)->wait();
+    echo get_class($article), PHP_EOL;
+    if ($article instanceof eLife\ApiSdk\Model\ArticleVoR) {
+        echo 'References: ', count($article->getReferences()), PHP_EOL;
```
Comment (thewilkybarkid) [inline scripts/load_all_articles.php]: Might be better to use authors, applies to both VoR and PoA.

**Comment by giorgiosironi** [comment]
Running this latest commit, which loads all article versions, is taking a while...

**Comment by giorgiosironi** [comment]
```
Invalid articles (not served): 5
Valid articles (served): 3518
Valid versions (served): 5675
$articleIds: 3518
$versions (indexed by article): 3518
Total versions (served): 5675
```
Trying to understand which 5, may not be related to the schema update.

**Comment by giorgiosironi** [comment]
The 5 articles are from https://elifesciences.atlassian.net/browse/PI-332 which is a different issue, we're good here.

**Comment by giorgiosironi** [comment]
Going to merge this to have the script running in the IIIF branch.

File: scripts/load_all_articles.php [hunk 2]
```diff
@@ -50,6 +50,26 @@
     //echo "Count: $articlesCount", PHP_EOL;
     //echo 'Memory: ', memory_get_usage(true), ' bytes', PHP_EOL;
 }
+$versionsByArticle = [];
+$versionsCount = 0;
+foreach ($articleIds as $id) {
+    $history = $articles->getHistory($id)->wait();
```
Comment (thewilkybarkid) [inline scripts/load_all_articles.php]: If you cycle through the histories then `wait()` on all of them it'd be faster.

File: scripts/load_all_articles.php [hunk 3]
```diff
@@ -50,6 +50,26 @@
...
+    foreach ($history->getVersions() as $article) {
+        $versionsByArticle[$id][] = $article->getVersion();
+        ++$versionsCount;
+    }
+}
+$totalVersions = 0;
+foreach ($versionsByArticle as $id => $versions) {
+    foreach ($versions as $version) {
+        $article = $articles->get($id, $version)->wait();
```
Comment (thewilkybarkid) [inline scripts/load_all_articles.php]: Same here.

File: scripts/load_all_articles.php (hunk 2 above)
Comment (giorgiosironi) [reply scripts/load_all_articles.php]: Last time there were some problems on loading too many responses at once, let's try.

**Pull request merged**